File: Multiplication.cpp
#include<iostream>
#include<iomanip>
using namespace std;

int main(){
    int num;
    int val;
    int result;
    cout<<"enter the number whose multiplication table you want :"<<endl;
    cin>>num;
    cout<<endl;
    

    for(val=1;val<=10;val++){
        result = val*num;
        cout<<num<<" x "<<setw(4)<<val<<" = "<<setw(4)<<result<<endl;
        
    }
    return 0;
}

File: Multiplication.py
#include <iostream>
using namespace std;
int main()
{
    int val;
    cout << "Enter the number " << endl;
    cin >> val;

    for (int i = 1; i <= 10; i++)
    {
        cout << val << " X " << i << " = " << val * i << endl;
    }
    return 0;
}
